examples/components/data_table/selectable_usage.html

```diff
@@ -10,14 +10,14 @@
 </md-toolbar>
 <md-data-table layout-fill [selectable]="true" (onSelectableChange)="change($event)">
   <thead>
-  <tr md-data-table-header-row>
+  <tr md-data-table-header-selectable-row>
     <th class="md-text-cell">Material</th>
     <th>Quantity</th>
     <th>Unit price</th>
   </tr>
   </thead>
   <tbody>
-  <tr md-data-table-row *ngFor="#material of materials" [selectable-value]="material.id">
```

Comment:
Do we want `md-data-table-selectable-row` or just a `md-data-table-row` with a `selectable` property? My instinct is the latter, but I could be convinced either way.